test/unit/slurm/test_slurm_config_generator/test_generate_slurm_config_files_memory_scheduling/expected_outputs/pcluster/slurm_parallelcluster_efa_partition.conf

```diff
@@ -1,7 +1,7 @@
 # This file is automatically generated by pcluster
 
-NodeName=efa-st-efa-c5n-[1-1] CPUs=36 RealMemory=186777 State=CLOUD Feature=static,c5n.18xlarge,efa-c5n,efa
-NodeName=efa-dy-efa-c5n-[1-4] CPUs=36 RealMemory=186777 State=CLOUD Feature=dynamic,c5n.18xlarge,efa-c5n,efa
+NodeName=efa-st-efa-c5n-[1-1] CPUs=36 RealMemory=186777 State=CLOUD Feature=static,c5n.18xlarge,efa-c5n,efa Weight=1
+NodeName=efa-dy-efa-c5n-[1-4] CPUs=36 RealMemory=186777 State=CLOUD Feature=dynamic,c5n.18xlarge,efa-c5n,efa Weight=1001
```

Comment:
Here and below.
Why you added this to an existing test ?

Do we want to test everything everywhere ?
Having simple more focused tests would make it more clear what we are testing and if something fails ..what failed and why.

Reply:
Since the parameter is always defined in the compute resource, the equivalent Slurm node parameter is always rendered in all places where the rendering of the Slurm node configuration is executed, so unfortunately all existing rendering tests are affected.

To avoid this we should include a piece of logic in the rendering only for testing purposes, which I personally would avoid.